tests/test_cloud_for_customers.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for CloudForCustomers source class"""
+
+from viadot.config import get_source_credentials as get_credentials
+from viadot.sources.cloud_for_customers import CloudForCustomers
+import pandas as pd
+
+
+CONTACT_ENDPOINT = "ContactCollection"
+cursor_field = "EntityLastChangedOn"
+cursor_value = "2022-10-30T00:00:00Z"
+cursor_filter = f"{cursor_field} ge datetimeoffset'{cursor_value}'"
+filter_params = {"$filter": cursor_filter}
+
+
+def test_cloud_for_customers_to_df(TEST_C4C_API_URL):
+    credentials_secret = "aia-c4c-qa"
+    print(credentials_secret)
+    credentials = get_credentials(credentials_secret)
+    print(credentials)
```

Comment:
remove... if we were running the tests in CI these credentials would already be compromised as they would show up in logs...

Reply:
True, sorry for this one. I will review my own PR's after pushing new changes to avoid completely this kind of silly mistaked.